Common/UnitDefinitions/LeakRate.json

```diff
@@ -0,0 +1,52 @@
+{
+    "Name": "LeakRate",
+    "BaseUnit": "PascalQubicMeterPerSecond",
+    "XmlDocSummary": "The leakrate is a measure of the units of volume or mass leaking from a body.",
+    "BaseDimensions": {
+      "M": 1,
+      "L": 2,
+      "T": -3
+    },
+    "Units": [
+        {
+            "SingularName": "PascalQubicMeterPerSecond",
+            "PluralName": "PascalQubicMeterPerSecond",
+            "FromUnitToBaseFunc": "{x}",
+            "FromBaseToUnitFunc": "{x}",
+            "XmlDocRemarks": "https://de.wikipedia.org/wiki/Leckrate#Formelzeichen_und_Ma%C3%9Feinheit",
```

Comment:
Are there English references we can link to? I didn't find an English wiki though.